bot/decorators.py

```diff
@@ -236,3 +238,31 @@ async def wrapped(*args, **kwargs) -> t.Any:
             return await func(*args, **kwargs)
         return wrapped
     return decorator
+
+
+def ensure_duration_in_future(duration_arg: function.Argument) -> t.Callable:
+    """
+    Ensure the duration argument is in the future.
+
+    If the condition fails, a warning is sent to the invoking context.
```

Comment:
Imperative mood rather than passive voice is more consistent.
```suggestion
    If the condition fails, send a warning to the invoking context.
```

Reply:
resolved in 574adc3